.github/labels.yml

```diff
@@ -0,0 +1,33 @@
+# This file is used by Auto Label PRs action.
```

Comment:
Suggest to rename this file to `labeler.yaml`. Please also modify the action using this file.

https://github.com/actions/labeler?tab=readme-ov-file#usage